config/routes.rb

```diff
@@ -11,8 +11,8 @@
   namespace :api do
     namespace :v1 do
       resources :brands, only: [:index, :show]
-      resources :pizzas, only: [:show, :new]
+      resources :pizzas, only: [:show, :new, :create]
```

Comment:
for pizzas `create` I would recommend nesting under the brand, take a look at my comment above on the form, so that the path reads `/api/v1/brands/:brandId/pizzas` and you will not have to pass the brand id around as props